style.css

```diff
@@ -422,6 +501,238 @@ form textarea:focus {
+}
+
+#textarea {
+  background-image: url(./imges/Icon\ -\ Resizer\ Handle.svg);
+  background-repeat: no-repeat;
+  background-position: left 325px, 200px 385px, right 280px;
+  background-size: 130px;
+}
+
+#mobile-menu {
+  height: 85vh;
+  border-bottom-right-radius: 2em;
+  border-bottom-left-radius: 2em;
+  margin-bottom: 2em;
+  mix-blend-mode: normal;
+  box-shadow: 0 1em 1em rgba(37, 47, 137, 0.08);
+}
+
+#mobile-menu > ul {
+  clear: right;
+  padding-left: 0;
+}
+
+.menu-link {
+  display: block;
+  text-decoration: none;
+  margin: 1em 1em;
+  color: #3a4a42;
+  font-size: x-large;
+  font-weight: 700;
+}
+
+.hidden {
+  display: none;
+}
+
+.social-media-icons li:hover {
+  transform: scale(1.3);
+}
+
+@media (min-width: 768px) {
+  /* Standard CSS rules */
+  html {
+    box-sizing: border-box;
+  }
+
+  header > a,
+  header > img {
+    display: none;
+  }
+
+  header {
+    padding: 0;
+  }
+
+  #headline {
+    background-image: url("./imges/Desktop/Header-llustration.png");
+    background-size: 106% 16%;
+  }
+
+  .my-placeholder {
+    grid-template-columns: repeat(3, 1fr);
+    height: 372px;
+    width: 229px;
+  }
+
+  nav {
+    margin-top: -1em;
+  }
+
+  .middle {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  nav a {
+    text-decoration: none;
+    color: #172b4d;
+    font-size: large;
+    font-weight: 550;
+  }
+
+  nav a:focus {
+    color: #008552;
+  }
+
+  .only-desktop {
+    display: block;
+  }
+
+  .tittleContents > h2 {
+    font-size: 275%;
+  }
+
+  .tittleContents > h1 {
+    font-size: 275%;
+  }
+
+  .primary > h1 {
+    font-size: 275%;
+  }
+
+  .img1-aboutme > p {
+    font-size: 150px;
+  }
+
+  .grid {
+    width: 95%;
+    margin: 2em;
+    grid-template-columns: repeat(3, minmax(300px, 1fr));
+    gap: 5px;
+  }
+
+  #aboutme {
+    grid-template-columns: repeat(3, 1fr);
+    padding-bottom: 11px;
+    margin-bottom: 5%;
+    border: solid 1px #d0d9d4;
+    background-color: #ebf0ee;
+    border-radius: 10px 10px 10px 10px;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+    width: 100%;
+  }
+
+  .img1-aboutme {
+    background-image: url("./imges/Desktop/about-me-illustration1.png");
+    background-size: 100px;
+    background-position-y: -15px;
+  }
+
+  .img1-aboutme + h2 {
+    padding: 0;
+  }
+
+  .supporting-text > p {
+    font-size: 150%;
+  }
+
+  .skills-card {
+    margin: 22px;
+    display: grid;
+    gap: 24px;
+    justify-items: center;
+  }
+
+  #about-me > p {
+    text-align: center;
+    padding: 0 5em;
+    margin: 0;
+  }
+
+  .img2-aboutme {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .mobile-only {
+    display: none;
+  }
+
+  .disabled-btn {
+    padding-top: 2em;
+    padding-bottom: 7em;
+  }
+
+  .section-contact {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .section-contact h2 {
+    width: 40%;
+    text-align: left;
+  }
```

Comment:
- [ ] Kindly make sure your contact section sticks to the Figma design. See the difference bellow 🙃 😉 

Your design

<img width="845" alt="Screen Shot 2022-11-12 at 00 47 19" src="https://user-images.githubusercontent.com/74501523/201441459-938b52ab-5374-4b02-bf43-62bde9d34fc9.png">

Figma design

<img width="719" alt="Screen Shot 2022-11-12 at 00 46 37" src="https://user-images.githubusercontent.com/74501523/201441491-71d9dfd1-c620-48e4-9494-ba2b5750a71d.png">

Reply:
- [ ] Kindly follow the design provided by Figma styling your elements related to the previous data. Your UI will look better to recruiters and hiring managers.

Figma design

<img width="676" alt="Screen Shot 2022-11-12 at 8 33 14 PM" src="https://user-images.githubusercontent.com/47673311/201498762-e978079c-aaf8-4e15-affd-8b8632ae1070.png">

Your design

<img width="1278" alt="Screen Shot 2022-11-12 at 8 32 44 PM" src="https://user-images.githubusercontent.com/47673311/201498769-888160b9-f6ad-4a5d-91fd-6ad7a1247906.png">